webapp/src/main/java/life/qbic/views/login/LoginHandler.java

```diff
@@ -57,24 +57,36 @@ public void handle(LoginLayout loginView) {
   }
 
   private void initFields() {
-    clearMessages();
-  }
-
-  private void clearMessages() {
-    registeredLoginView.clearErrors();
-    registeredLoginView.clearInformation();
+    clearNotifications();
   }
 
   private void showInvalidCredentialsError() {
-    registeredLoginView.showError(INCORRECT_USERNAME_OR_PASSWORD);
+    showError(INCORRECT_USERNAME_OR_PASSWORD);
   }
 
   private void showEmailConfirmationInformation() {
-    registeredLoginView.showInformation(EMAIL_CONFIRMATION_SUCCESS);
+    showInformation(EMAIL_CONFIRMATION_SUCCESS);
   }
 
   private void showEmailConfirmationReminder() {
-    registeredLoginView.showInformation(EMAIL_CONFIRMATION_REMINDER);
+    showInformation(EMAIL_CONFIRMATION_REMINDER);
+  }
+
+  public void clearNotifications() {
+    registeredLoginView.notificationLayout.setVisible(false);
+    registeredLoginView.notificationLayout.removeAll();
+  }
+
+  public void showError(ErrorMessage errorMessage) {
+    registeredLoginView.notificationLayout.add(
+        new ErrorMessage(errorMessage.title(), errorMessage.message()));
+    registeredLoginView.notificationLayout.setVisible(true);
```

Comment:
why do you create a new message? You could also just forward the error message from the constructor. 
```suggestion
    registeredLoginView.notificationLayout.add(errorMessage);
    registeredLoginView.notificationLayout.setVisible(true);
```

Reply:
If you leave the layout visible you dont need to make it visible again here

Reply:
Good point 👍